### ISSUE TITLE
sql is in the list - but it says it's not support

### ISSUE THREAD

**Issue by NorfeldtAbtion** [zakhenry/embedme#42]
This is a genius package! 🚀

I tried a dry run of an sql snippet but it gave me:

```
README.md#L65-L67 Unsupported file extension [sql], supported extensions are txt, ts, js, scss, rust, java, cpp, c, html, xml, md, yaml, json, json5, py, bash, sh, go, objectivec, php, cs, swift, rb, kotlin, scala, cr, puml, mermaid, cmake, proto, skipping code block
```

**Comment by zakhenry** [comment]
Hi @NorfeldtAbtion thanks for the kind words and for bringing this to my attention. It looks like the last pull request failed to deploy for some reason. I'll dig in to it and report back 👍

**Comment by zakhenry** [comment]
Hi @NorfeldtAbtion this should work now, sorry for the delay there was some weirdness in Travis CI not publishing to NPM which is resolved now (`embedme@1.18.1`)

**Comment by NorfeldtAbtion** [comment]
There is absolutely nothing to be sorry for! Awesome project and very swift response! 🎉
